packages/blockchain-link/src/workers/ripple/index.ts

```diff
@@ -36,50 +52,43 @@ const transformError = (error: any) => {
 };
 
 const getInfo = async (request: Request<MessageTypes.GetInfo>) => {
-    const api = await request.connect();
-    const info = await api.getServerInfo();
+    const client = await request.connect();
+    const response = await client.request({
+        command: 'server_info',
+    });
 
     // store current ledger values
-    RESERVE.BASE = api.xrpToDrops(info.validatedLedger.reserveBaseXRP);
-    RESERVE.OWNER = api.xrpToDrops(info.validatedLedger.reserveIncrementXRP);
+    if (response.result.info.validated_ledger != null) {
+        RESERVE.BASE = xrpToDrops(response.result.info.validated_ledger.reserve_base_xrp);
+        RESERVE.OWNER = xrpToDrops(response.result.info.validated_ledger.reserve_inc_xrp);
```

Comment:
Should be `1` 
<img width="1143" alt="Screenshot 2025-03-08 at 11 24 59" src="https://github.com/user-attachments/assets/d6ce3387-7fe5-4a63-9540-e55ca0c71a45" />

Reply:
<img width="423" alt="Screenshot 2025-03-08 at 11 26 33" src="https://github.com/user-attachments/assets/252f2d95-1049-494e-b637-8453031ef0c2" />